src/actions/instagram/chats.ts

```
@@ -0,0 +1,215 @@
+
+    const { data } = await response.json();
+
+    const messages = data.map((message: InstagramMessage) => {
+      const formattedMessage = {
+        id: message.id,
+        text: message.message || "",
+        sender: message.from.username || message.from.id,
+        timestamp: message.created_time,
+      } as Message;
+
+      if (message.attachments && message.attachments.data && message.attachments.data.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)